redux/reducers/User.ts

```diff
@@ -22,3 +22,6 @@ export const User = createSlice({
 
 export const {logIn, resetToInitialState, updateToken} = User.actions;
 export default User.reducer;
+
+export const selectUser = (state: {user: {[key: string]: string}}) =>
```

Comment:
Here is how to properly type the redux state: https://react-redux.js.org/using-react-redux/usage-with-typescript